src/FubuMVC.Core/ServiceBus/Configuration/ChannelGraph.cs

```diff
@@ -112,6 +111,20 @@ public Uri ReplyChannelFor(string protocol)
             return _replyChannels[protocol];
         }
 
+        public List<string> GetAcceptedContentTypesForChannel(Uri channelUri)
```

Comment:
Can you do this w/ just some yield return calls? It's a little hokey to be adding all the nulls into the list just to filter them out.